cal/schema.py

```diff
@@ -36,6 +38,31 @@ def to_json(self):
             "user_id": self.user_id
         }
 
+    @staticmethod
+    def fuzzy_contains(event):
+        time_fuzz = dt.timedelta(days=3)
+
+        start_match = (Event.start < event.start + time_fuzz) & \
+                      (Event.start > event.start - time_fuzz)
+        end_match = (event.end==None) or \
+                    ((Event.end == None) | \
+                        ((Event.end < event.end + time_fuzz) 
+                            & (Event.end > event.end - time_fuzz)))
```

Comment:
Hm... this looks pretty hard to read. Is there a better way to structure this? Maybe: 
```Python
query = (Event.start < event.start + time_fuzz) & \
        (Event.start > event.start - time_fuzz)
if end_match is not None:
    query &= (Event.end is None) | ((Event.end < event.end + time_fuzz) & \
                                     Event.end > event.end + time_fuzz))
```

Also, generally it's `x is None` instead of `x == None`